test/Pool.test.js

```diff
@@ -1,17 +1,17 @@
-var assert = require('assert');
-var {Promise} = require('../src/Promise');
-var Pool = require('../src/Pool');
-var tryRequire = require('./utils').tryRequire
+var assert = require("assert");
```

Comment:
I see you have different coding style configured in your IDE 😅 . We should really setup a linter for the project, let me see if I can set up Biomejs (if so we can run it after merging this PR, otherwise we'll have a lot of merge conflicts). 

Is it possible to revert the coding style changes, so I can see which unit tests are added/updated?

Reply:
Yes sorry I switched Editors recently.

Reply:
I reverted back to the upstream master and applied my changes to the `abort handler` test suite. there are a couple formatting changes that I'll clean up. but the tests where largely re written to wait until the abort operation concludes to assert on the pool status. Which makes it a larger diff.